runtime/src/accounts_index.rs

```diff
@@ -41,7 +41,7 @@ pub type AccountMap<K, V> = BTreeMap<K, V>;
 
 type AccountMapEntry<T> = Arc<AccountMapEntryInner<T>>;
 
-pub trait IsCached {
+pub trait IsCached: 'static + Clone + Debug {
```

Comment:
I was also expecting to see where `'static + Clone + Debug` was _removed_ from current uses of `IsCached` as well. Is that meant for a later PR then?

Reply:
Yes, it's just messy and not required yet. Really, later I have to add, sync + send. That change will now be one line. And, new code written against these types will now not have to string along ~7 traits. I'll clean that all up at some point.

Reply:
but your expectation was reasonable...